### PR TITLE
fix : search box scroll

### PR DESCRIPTION
**Pull request by WiseyXD**
Added a scroll to search scroll through all the recommended commands and the scroll is responsive for all devices (sizes)

https://github.com/user-attachments/assets/c5aa258e-a37e-4ef4-97fb-f031881c005e

### CODE REVIEW

File: package-lock.json
Comment (shubhdevelop) [inline package-lock.json:1]: why we pushing changes in package-lock.json??

**Comment by shubhdevelop** [comment]
nit: Add screencast for of desktop view scroll in the PR description,
create an issue and link this PR to that

File: package-lock.json
Comment (WiseyXD) [reply package-lock.json:1]: is this a wrong practice ?

Comment (shubhdevelop) [reply package-lock.json:1]: yes, ideally lock files doesn't change until unless a package is added/removed or we are changing the versioning of the packages

**Comment by WiseyXD** [comment]
Fixed issue : https://github.com/DiceDB/playground-web/issues/33#issue-2564437196

**Comment by lucifercr07** [comment]
@WiseyXD please resolve conflicts.

**Comment by shubhdevelop** [comment]
> Fixed issue : [#33 (comment)](https://github.com/DiceDB/playground-web/issues/33#issue-2564437196)

you mention the issue,  didn't link this. Put this inside the PR description

**Pull request closed without merging**